packages/visualizations/src/components/Table/Table.svelte

```diff
@@ -14,18 +14,23 @@
     const tableId = `table-${generateId()}`;
 
     $: ({ value: records } = data);
-    $: ({ columns, title, subtitle, description, source } = options);
+    // FIXME: Eslint is in conflict with prettier
+    // eslint-disable-next-line object-curly-newline
+    $: ({ columns, title, subtitle, description, source, unstyled } = options);
+    $: defaultStyle = !unstyled;
 </script>
 
-<div class="container">
-    <div class="header">
-        {#if title}
-            <h3>{title}</h3>
-        {/if}
-        {#if subtitle}
-            <p>{subtitle}</p>
-        {/if}
-    </div>
+<div class="ods-dataviz-sdk-table" class:ods-dataviz-sdk-table--default={defaultStyle}>
+    {#if title || subtitle}
+        <div class="dataviz-header">
```

Comment:
I think `header` is fine, it's scoped. I like keeping scoped names simple, it reads more obviously "I'm scoped"

Reply:
Done in the next PR because I moved the style 😅